GeoLib/GEOObjects.cpp

```diff
@@ -90,29 +91,21 @@ bool GEOObjects::appendPoint(Point* point, std::string const &name, std::size_t&
 const std::vector<Point*>* GEOObjects::getPointVec(const std::string &name) const
 {
 	int idx = this->exists(name);
```

Comment:
we could change the ```exists()```-method to return ```std::numeric_limits::max()``` if the name does not exits, this would save some conversions here and other methods where this is checked.